docs/baseband/tutorials/getting_started.rst

```diff
@@ -83,39 +83,37 @@ decoded data array.  Reading files with multiple threads and channels will
 produce 3-dimensional arrays.
 
 If you want to know the shape of a single complete sample - the set of samples
-from all available threads and channels for a given point in time - it is
+from all available threads, channels, etc., for a given point in time - it is
 accessible through::
 
     >>> fh.sample_shape
     SampleShape(nthread=8)
 
 By default, dimensions of length unity are removed from the sample shape.  To
-retain them, set the ``fh.squeeze`` attribute to `False`:
+retain them, we can pass ``squeeze=False`` to `~baseband.vdif.open`:
 
-    >>> fh.squeeze = False
-    >>> fh.sample_shape              # Sample shape now keeps channel dimension.
+    >>> fhns = vdif.open(SAMPLE_VDIF, 'rs', squeeze=False)
+    >>> fhns.sample_shape    # Sample shape now keeps channel dimension.
     SampleShape(nthread=8, nchan=1)
-    >>> d2 = fh.read(12)
-    >>> d2.shape                     # Decoded data has channel dimension.
+    >>> d2 = fhns.read(12)
+    >>> d2.shape             # Decoded data has channel dimension.
     (12, 8, 1)
-    >>> fh.squeeze = True            # Set back to `True` for rest of section.
+    >>> fhns.close()
 
-The ``squeeze`` attribute defaults to `False` for all stream readers and
-writers, but this behaviour can be overridden by passing ``squeeze=False`` to
-``open``
+``squeeze`` cannot be changed for an already-open file reader.
```

Comment:
I don't think this is necessary, as it is generally true that nothing can be changed in a file handle (I know, inconsistent with another comment....)